FlowerinoPlugin/src/org/flowerplatform/flowerino_plugin/command/UpdateSourceFilesCommand.java

```diff
@@ -0,0 +1,71 @@
+package org.flowerplatform.flowerino_plugin.command;
+
+import static org.flowerplatform.flowerino_plugin.FlowerPlatformPlugin.log;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.flowerplatform.flowerino_plugin.FlowerPlatformPlugin;
+import org.flowerplatform.flowerino_plugin.SourceFileDto;
+import org.flowerplatform.tiny_http_server.IHttpCommand;
+
+import processing.app.BaseNoGui;
+import processing.app.Editor;
+
+/**
+ * TODO: This is currently unused; need to see if we really need it; if not => we can remove.
+ * @author Claudiu Matei
+ *
+ */
+public class UpdateSourceFilesCommand extends ArrayList<SourceFileDto> implements IHttpCommand, List<SourceFileDto> {

```

Comment:
logica exista in cealalta. Cred ca a fost reimplementata. De pus cea mai buna varianta.